[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/fomightez/Unipressed-binder/main?urlpath=%2Flab%2Ftree%2FUnipressed_examples_running_via_MyBinder.ipynb)

*tl;dr:*  
Click any `launch binder` badge on this page to run UniProt queries inside your browser.

------


# Unipressed-binder
Unipressed examples that run in MyBinder-served sessions to illustrate use of Unipressed to query UniProt.

-----------------------

There's a Python package, [Unipressed](https://multimeric.github.io/Unipressed/), by Michael Milton ([@multimeric](https://twitter.com/multimeric)) that allows programmatic access query UniProt's new REST API.   
If you use a launch badge on this page a Jupyter notebook will be presented where you can run work through running code to demonstrate the use of Unipressed to query UniProt's database and get useful information related to your biological molecules of interest.

I, Wayne, am not associated with Unipressed development. For that information, see here about [Unipressed](https://multimeric.github.io/Unipressed/).    
This repo is only set up to make it convenient to try queries of UniProt via Unipressed in MyBinder-served sessions.  
The idea is that you can then adapt these examples to your own queries and perform them without even installing anything on your own machine, provided the computational resources in the MyBinder-served session are sufficient. If not, the examples should guide you in using the code in other places or give you a place to troubleshoot with less demanding tasks.

------


### Related use of the package

- I use this package to add information from UniProt to reports made for proteomically-identified human complexes listed in hu.MAP 3.0 data, see [my humap3-binder repo](https://github.com/fomightez/humap3-binder) for more information. 
- I have done much the same as above for hu.MAP 2.0 data, see [my humap2-binder repo](https://github.com/fomightez/humap2-binder).

-----

### Related software packages or libraries:

- [R-based: 'ginmappeR: an unified approach for integrating gene and protein identifiers across biological sequence databases' by Sola et al 2024](https://academic.oup.com/bioinformaticsadvances/article/4/1/vbae129/7745345?login=false), package: [ginmapppeR at bioconductor](https://bioconductor.org/packages/release/bioc/html/ginmappeR.html)

------

JupyterLab interface: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/fomightez/Unipressed-binder/main?urlpath=%2Flab%2Ftree%2FUnipressed_examples_running_via_MyBinder.ipynb)  
Jupyter Notebook 7+:  [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/fomightez/Unipressed-binder/main?urlpath=%2Ftree%2FUnipressed_examples_running_via_MyBinder.ipynb)
